src/containers/SubjectPage/SubjectContainer.jsx

```diff
@@ -321,12 +293,20 @@ const SubjectPage = ({
 SubjectPage.getDocumentTitle = getDocumentTitle;
 
 SubjectPage.willTrackPageView = (trackPageView, currentProps) => {
-  const { data } = currentProps;
-  if (data?.subject?.topics?.length > 0) {
+  const { data, loading } = currentProps;
+  if (!loading && data?.subject?.topics?.length > 0) {
     trackPageView(currentProps);
   }
 };
 
+SubjectPage.getDimensions = props => {
+  const { data, topics } = props;
+  const topicPath = topics.map(t =>
+    data.subject.allTopics.find(topic => topic.id === t),
+  );
+  return getAllDimensions({ subject: data.subject, topicPath });
```

Comment:
Skulle gjerne hatt article i dimensjonene som sendes inn her. Men det krever at det gjøres et ekstra kall for å hente artikkelen som ligg bak siste leddet i topicPath. Dataene hentes jo via graphql i Topic, så alternativt kunne det vore sendt en pageview her for subject og en anna for topic inni Topic.

Reply:
Jeg har prøvd og lett, men finner ingen god løsning for å sende inn data fra graphql-spørringen i Topic til trackPageView. Har du noen peiling på hvordan det kan gjøres?

Reply:
Antar det enkleste er å lage en TopicWrapper som du kaller i stedet for Topic, og som gjør henting av data. Denne igjen kan ha en Topic som tar inn data som prop og som har withTracker og dermed kan tracke at topicen vises.